test_install/test_install.cpp

```diff
@@ -357,13 +357,13 @@ int main(int, char **)
                            .with_reduction_factor(1e-10)
                            .on(refExec);
 
-        auto rel_res = gko::stop::RelativeResidualNorm<>::build()
-                           .with_tolerance(1e-10)
-                           .on(refExec);
+        auto rel_res =
+            gko::stop::RelativeResidualNorm<>::build().with_tolerance(1e-10).on(
+                refExec);
 
-        auto abs_res = gko::stop::AbsoluteResidualNorm<>::build()
-                           .with_tolerance(1e-10)
-                           .on(refExec);
+        auto abs_res =
+            gko::stop::AbsoluteResidualNorm<>::build().with_tolerance(1e-10).on(
+                refExec);
```

Comment:
It seems to be formatted not well, but my local formatted result is the same as it

Reply:
I wonder why this was reformatted like this. The existing code is actually easier to read and looks better.

Reply:
Yes, this is pretty unfortunate. Luckily, this is only part of our install test, which we only rarely need to modify anyways.

Reply:
Yes, when reviewing this I also thought the previous version looked better, but that's the usual `clang-format` shenaningans.